src/main/java/uk/co/eclipse/billing/ngtesting/https/libs/RobotKeyEvents.java

```diff
@@ -23,28 +26,60 @@ public RobotKeyEvents(WebDriver webDriver){
         }
     }
 
+    public interface User32 extends W32APIOptions {
+        User32 instance = (User32) Native.loadLibrary("user32", User32.class, DEFAULT_OPTIONS);
+        boolean ShowWindow(WinDef.HWND hWnd, int nCmdShow);
+        boolean SetForegroundWindow(WinDef.HWND hWnd);
+        boolean SetFocus(WinDef.HWND hWnd);
+        WinDef.HWND FindWindow(String winClass, String title);
+        int SW_SHOW = 1;
+    }
+
+    public void SetActiveWindow(){
+        wait3Second();
+        User32 user32 = User32.instance;
+        WinDef.HWND hWnd = user32.FindWindow(null, "Open");
+        user32.ShowWindow(hWnd, User32.SW_SHOW);
+        user32.SetForegroundWindow(hWnd);
+        user32.SetFocus(hWnd);
+        logger.info("Active window is set up");
+    }
+
     public void typeText(String string){
         try {
-                int length = string.length();
+            wait3Second();
+            SetActiveWindow();
+            wait3Second();
+            int length = string.length();
                 for (int i = 0; i < length; i++)
```

Comment:
Var length is never used instead of this block. So you can use here string.length()